src/entities/Order.java

```diff
@@ -0,0 +1,65 @@
+package entities;
+
+import enums.OrderStatusEnum;
+
+import java.util.List;
+
+public class Order extends BaseEntity<Long>{
```

Comment:
You may need an order tax relationship and should be a list.

You also may want to store the amount the client paid as well.

Reply:
I have an order tax entity, in the OrderTax branch, it's in the Adding Order Tax pr. I've already created the order entity and pr  before creating this order branch and pr (my mistake), so thats why it doesnt show the order tax list

I'll add the amount the client paid